global.d.ts

```diff
@@ -58,6 +58,16 @@ declare global {
        * - my test
        *   - TODO(NODE-XXXX): Feature implementation impending!
        * ```
+       * 
```

Comment:
Should this documentation live in our testing readme instead of the global types file?

Reply:
Nevermind, I see you added it there as well

Reply:
Yeah, Neal reminded me that we have it in the types even though the types aren't quire working yet for test